src/components/TransferForm.tsx

```diff
@@ -2,7 +2,7 @@ import * as copy from 'copy-to-clipboard'
 import * as React from 'react'
 
 import { TransferProps } from '@containers/TransferForm'
-import { formAlert } from '@helpers/alert'
+import { formAlert } from '@helpers/alert';
```

Comment:
this should be removed by linting I would have thought? would be nice to keep the file from having any changes

Reply:
the linter didn't pick it up, so I removed it.